Arduboy.cpp

```diff
@@ -682,14 +687,12 @@ unsigned char* Arduboy::getBuffer()
 
 boolean Arduboy::pressed(uint8_t buttons)
 {
- uint8_t button_state = getInput();
- return (button_state & buttons) == buttons;
```

Comment:
Did you compare compile size for this change? If it's not smaller I like the clarity of the previous code.